Zend/tests/type_declarations/confusable_type_warning.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Warnings for confusable types
+--FILE--
+<?php
+
+function test1(integer $x) {}
```

Comment:
This suggests `or "use" to suppress this warning in %s on line %d`, which will cause its own compile warning.  

```
php > use integer;
Warning: The use statement with non-compound name 'integer' has no effect in php shell code on line 1
```

- Could add unit test with `use foo as integer` not warning

Reply:
I've adjusted the warning to only mention `use` if inside a namespace.